Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
@jrtibbetts Any downstream effects of taking a dependency on CocoaPods in our example app?

Reply:
This just sets a minimum CocoaPods version. I can take it out.